test/functional/scan/tests/test-scan-inclusive.hpp

```diff
@@ -116,10 +119,13 @@ TYPED_TEST_P(ScanFunctionalTest, ScanInclusive)
 
 TYPED_TEST_P(ScanFunctionalTest, ScanInclusiveInplace)
 {
-  using EXEC_POLICY      = typename at<TypeParam, num<0>>::type;
-  using WORKING_RESOURCE = typename at<TypeParam, num<1>>::type;
-  using OP_TYPE          = typename at<TypeParam, num<2>>::type;
+  using EXEC_POLICY      = typename camp::at<TypeParam, camp::num<0>>::type;
+  using WORKING_RESOURCE = typename camp::at<TypeParam, camp::num<1>>::type;
+  using OP_TYPE          = typename camp::at<TypeParam, camp::num<2>>::type;
```

Comment:
Specifying the full camp scope is probably a good idea. I'll make a note to change this in my own test PRs.

Reply:
Yes, it helps make clear what's part of RAJA and what is not.  Also, we're trying to minimize the use of using directives in header files.